redfish-core/include/registries/privilege_registry.hpp

```diff
@@ -1421,6 +1421,30 @@ const static auto& putPowerDistributionMetrics =
 const static auto& deletePowerDistributionMetrics =
     privilegeSetConfigureComponents;
 
+// PowerSubsystem
+const static auto& getPowerSubsystem = privilegeSetLogin;
```

Comment:
Why the changes here?

Reply:
Done. I deleted it. @lkammath  Please pick the latest patchset.